tests/manage/pv_services/pvc_snapshot/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import os
+import urllib.request
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.helpers import helpers
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    tier1,
+    polarion_id,
+    bugzilla,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@polarion_id("OCS-3946")
+class TestPvcClonedStatusOfSmallSizeFiles(E2ETest):
+    """
+    Tests to verify restored PVC status bound or not if
+    files size are smaller.
+    """
+
+    @bugzilla("2039265")
+    def test_pvc_status_after_clone(
```

Comment:
All the setup is done inside the fucntion. All the part from the `class TestPvcClonedStatusOfSmallSizeFiles(E2ETest):` to `def test_pvc_status_after_clone(` is for setup test. Our best practice is to to all the setup like create pvc, create pod, create files in the setup phase and then the function should check just what the test is about like create the snapshot and verify it.
You can look at this test to understand https://github.com/shyRozen/ocs-ci/blob/master/tests/lvmo/test_lvm_snapshot_base.py
All the function in the setup are fixtures that are send to the test function.
If you need some clarification please contact me.